src/PHPJasper.php

```diff
@@ -215,6 +221,11 @@ public function execute($user = false)
 
         chdir($this->pathExecutable);
         exec($this->command, $output, $returnVar);
+
+        if (!is_null($this->tmpJsonFile)) {
```

Comment:
How about using the following code:

```
if ($this->tmpJsonFile !== null) {
```

Using the `is_null` function call will be slower than using value comparison.

The related refrence is available [here](http://php.net/manual/en/function.is-null.php).